documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -109,7 +109,11 @@ include::modules/configuring-mtv-operator.adoc[leveloffset=+2]
 [id="migrating-vms-web-console"]
 == Migrating virtual machines by using the {ocp} web console
 
-You can migrate virtual machines (VMs) to {virt} by using the {ocp} web console.
+You can migrate virtual machines (VMs) to {virt} by using the {ocp} web console to:
```

Comment:
I don't understand, "you can do X by using Y to: Z..." the prevoius "you can do X by using Y" makes more sense to me, what am i missing ?

Reply:
> I don't understand, "you can do X by using Y to: Z..." the prevoius "you can do X by using Y" makes more sense to me, what am i missing ?

+1

Reply:
@arik @yaacov 
Before, the text said you can migrate the VMs by using the OS web console.
Now, the text says where you are migrating them to --> (CNV or OSVirt), whatever you what to call it)., still using the OS web console.

If we were migrating them to RHV, we'd say "You cam migrate VMs to RHV by using the OS web console. "

But, if this is confusing,  I'll just drop "to {virt}" since the whole guide is about migrating to {virt}.